dts/bindings/memory-controllers/st,stm32-fmc-nor-psram.yaml

```diff
@@ -0,0 +1,194 @@
+      description: |
+        SRAM/NOR-Flash (read) timing register (FMC_BTRx).
+
+        If the EXTMOD is set (see control register FMC_BCRx), then
+        FMC_BTRx register is partitioned for write and read access.
+        That means, use this property to configure read accesses and
+        'st,timing-ext' to configure write accesses.
+
+        Expected fields, in order:
+
+        * ADDSET  - Address setup phase duration.
+                    Number of HCLK cycles to configure the duration of
+                    the address setup time. This parameter can be a value
+                    between Min_Data = 0 and Max_Data = 15.
+                    Note: Not used with synchronous NOR Flash memories.
+        * ADDHLD  - Address-hold phase duration.
+                    Number of HCLK cycles to configure the duration of
+                    the address hold time. This parameter can be a value
+                    between Min_Data = 1 and Max_Data = 15.
+                    Note: Not used with synchronous NOR Flash memories.
+        * DATAST  - Data-phase duration.
+                    Number of HCLK cycles to configure the duration of
+                    the data setup time. This parameter can be a value
+                    between Min_Data = 1 and Max_Data = 255.
+                    Note: Used for SRAMs, ROMs and asynchronous multiplexed
+                    NOR Flash memories.
+        * BUSTURN - Bus turnaround phase duration.
+                    Number of HCLK cycles to configure the duration of
+                    the bus turnaround. This parameter can be a value
+                    between Min_Data = 0 and Max_Data = 15.
+                    Note: Only used for multiplexed NOR Flash memories.
+        * CLKDIV  - Clock divide ratio (for FMC_CLK signal).
+                    Period of CLK clock output signal, expressed in number of
+                    HCLK cycles. This parameter can be a value
+                    between Min_Data = 2 and Max_Data = 16.
+                    Note: Not used for asynchronous NOR Flash, SRAM or ROM
+                    accesses.
+        * DATLAT  - Data latency for synchronous memory.
+                    Number of memory clock cycles to issue to the memory
+                    before getting the first data.
+                    The value depends on the memory type as shown below:
+                    - It must be set to 0 in case of a CRAM
+                    - It is don't care in asynchronous NOR, SRAM or ROM accesses
+                    - It may assume a value between Min_Data = 2 and Max_Data = 17
+                      in NOR Flash memories with synchronous burst mode enable
+        * ACCMOD  - Access mode.
+                    See access mode defines
+                    in dt-bindings/memory-controller/stm32-fmc-nor-psram.h.
+
+    st,timing-ext:
+      type: array
+      required: false # required when 'EXTMOD' is enabled
+      default: [0xF, 0xF, 0xFF, 0xFF, 0x00] # reset state
```

Comment:
Please don't do it like this. The documentation I asked you to read says to put the justification for each default in the description for the property, not in a comment like this... I'm not trying to be a bad guy, but this happens all the time. People respond in GitHub comments, in YAML comments, all sorts of creative places that aren't the 'description' right below. Maybe the docs are not clear enough?